File: app/src/main/java/com/todolist/models/Task.java
package com.todolist.models;

import androidx.annotation.NonNull;

public class Task {
    private Integer id;
    private String definition;

    public Task(Integer id, String definition) {
        this.id = id;
        this.definition = definition;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getDefinition() {
        return definition;
    }

    public void setDefinition(String definition) {
        this.definition = definition;
    }

    @NonNull
    @Override
    public String toString() {
        return definition;
    }
}

File: Study/ToDoList/app/src/main/java/ru/brovkin/todolist/models/Task.kt
package ru.brovkin.todolist.models

import kotlinx.serialization.Serializable

@Serializable
class Task {
    var id: Int = 0
        get() = field
        set(value) {
            field = value
        }
    var definition: String? = null
        get() = field
        set(value) {
            field = value
        }
    constructor(definition: String?) {
        this.definition = definition
    }

    constructor(id: Int, definition: String?) {
        this.id = id
        this.definition = definition
    }

    override fun toString(): String {
        return definition.toString()
    }
}